cmd/podman/root.go

```diff
@@ -338,6 +339,10 @@ func rootFlags(cmd *cobra.Command, opts *entities.PodmanConfig) {
 
 	lFlags := cmd.Flags()
 
+	sshFlagName := "ssh"
+	lFlags.StringVar(&opts.SSHMode, sshFlagName, string(ssh.GolangMode), "define the ssh mode")
+	_ = cmd.RegisterFlagCompletionFunc(sshFlagName, completion.AutocompleteNone)
```

Comment:
Are there only a couple of sshmodes?  Shouldn't there be a command completion for them?